### PR TITLE
Initial pass through VOL_SharedCode formatting

### PR DESCRIPTION
**Pull request by jjbennett**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by jjbennett** [comment]
@lmeerkatz - I renamed the branch and will be using this branch now as a parent branch to the query locator work item. This first part is ready for review when you have a chance.
@kselvocki - This pull request should have no changes to behavior at all but since this is the shared code file, it would be good to run it through a quick regression. There will be a separate pull request that will have the actual fix to the work item - W-028654.
Thanks team!

File: src/classes/VOL_SharedCode.cls [hunk 1]
```diff
@@ -31,170 +31,216 @@
...
-        list<SelectOption> listSO = new list<SelectOption>();
-        listSO.add(new SelectOption('', ''));
-
-        // ensure valid date ranges
-        if (dtStart == null)
-            dtStart = system.today();
-        if (dtEnd == null)
-            dtEnd = system.today().addMonths(12);
-        dtEnd = dtEnd.addDays(1);
+    public static List<SelectOption> listSOVolunteerShiftsOfVolunteerJobIdFormat(
```
Comment (lmeerkatz) [inline src/classes/VOL_SharedCode.cls]: This one's only public, so if you wanted to change the method name ...

Comment (jjbennett) [reply src/classes/VOL_SharedCode.cls]: Done!

Comment (kselvocki) [reply src/classes/VOL_SharedCode.cls]: Ran partial regression--all is working as expected!

**Comment by kselvocki** [comment]
Ran partial regression--all is working as expected!

**Pull request merged**